pkg/pillar/Dockerfile

```diff
@@ -19,15 +19,16 @@ ARG DEV=n
 ENV BUILD_PKGS git gcc linux-headers libc-dev make linux-pam-dev m4 findutils go util-linux make patch wget \
```

Comment:
Shouldn't you be able to remove wget as well as curl?

Reply:
Yes, I only realized it was using wget later. Fixing.